packages/ecs/src/topic.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Topic, createTopic } from "./topic"
+
+describe("createTopic", () => {
+  it("creates a new topic with the given name", () => {
+    type TestType = {
+      mass: Number
+    }
+    const topic = createTopic<TestType>("testTopic");
+    expect(topic.name).toBe("testTopic");
+  })
+
+  it("adds events to it's list of events", () => {
+    type TestType = {
+      mass: Number
+    }
+    const topic: Topic<string, TestType> = createTopic<TestType>("testTopic");
+    topic.pushEvent({ mass: 17 });
+
+    for (const event of topic) {
```

Comment:
I get a really strange error on this line for some reason. I added `downlevelIteration: true` to each of the tsconfig.json files but can't seem to get rid of it. Are you seeing this same issue on your end?

Reply:
I ended up finding the solution - in our tsconfig.json at the `ecs/` level we were excluding .spec files which means my editor wasn't picking up any of the changes to the options.